doc/manuals/user/walkthrough_apiv2.md

```diff
@@ -121,12 +121,12 @@ command:
 ```
 # cd /dir/where/accumulator-server/is/downloaded
 # chmod a+x accumulator-server.py
-# ./accumulator-server.py --port 1028 --url /accumulate --host ::1 --pretty-print -v
+# ./accumulator-server.py --pretty-print -v
```

Comment:
Why this has been changed? Under my understanding the only changes we need in this file are the ones about Python 2.7. I mean to changes:

_In addition, it is recommended to use Python 2.7_

by something like this:

_In addition, it is recommended to use Python 3_

Reply:
Well, these are the default values for `port`, `url` and `host` variables, so I considered it wasn't needed to explicitly specify these default values as command line arguments. However ... we can preserve this string if you want.

Reply:
> Well, these are the default values for `port`, `url` and `host` variables, so I considered it wasn't needed to explicitly specify these default values as command line arguments. However ... we can preserve this string if you want.

I think they are useful to show the reader of the walk-through they are configurable so in case in its environment some of the default is not appropriated (e.g. port 1028 is being used by some existing service) he can know it can be changed.